modules/aws_eks/aws-eks.yaml

```diff
@@ -14,6 +14,10 @@ inputs:
     user_facing: false
     description: Opta Module name
     default: None
+  - name: cluster_name
```

Comment:
We are changing the cluster name in the terraform files from 
name     = "opta-${var.layer_name}"

to

 name     = var.cluster_name

Have you confirmed that for pre-existing clusters the name doesn't change from what its set to? To prevent cluster re-creation?

Reply:
yes, I am absolutely certain